### PR TITLE
Add primal.net logo to footer

### PR DESCRIPTION
**Pull request by orangesurf**
Adds Primal App logo (flattened layers, full white) to footer directing to [primal.net/mempool](https://primal.net/mempool)

image

image

As requested by @wiz

### CODE REVIEW

**Comment by orangesurf** [comment]
**Don't Merge**

Looks good on desktop and most mobiles (using chrome device presets) but list will roll over on some smaller devices where footer is shown

Galaxy S10/S10+, S20, iPhone11 Pro, iPhone 12/13 mini, iPhoneSE gen2,

image

**Comment by hunicus** [comment]
I couldn't get the icon to show without setting `viewBox="0 0 256 256"`. 

Working with orangesurf out-of-band to determine a solution to icon wrapping.

**Comment by orangesurf** [comment]
Screenshots

1920 Width
image

375 Width
(iPhone 11 Pro)
image

File: frontend/src/app/shared/components/global-footer/global-footer.component.scss [hunk 1]
```diff
@@ -97,6 +104,7 @@ footer .row.social-links a {
 
 footer .row.social-links svg {
   width: 20px;
+  margin: 10px 0 10px 0;
```
Comment (orangesurf) [inline frontend/src/app/shared/components/global-footer/global-footer.component.scss:107]: padding to ensure logos are spread out when they split onto two rows

File: frontend/src/app/shared/components/global-footer/global-footer.component.scss [hunk 2]
```diff
@@ -88,7 +88,14 @@ footer .row.link-tree {
...
 footer .row.social-links {
   text-align: center;
-  margin: 24px 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: fit-content;
+  margin: 0 auto; 
+
+  @media (max-width: 450px){
+    width: 250px;
```
Comment (orangesurf) [inline frontend/src/app/shared/components/global-footer/global-footer.component.scss:97]: For narrow screens set flex-box to a fixed width forcing a row of 4 on top and 3 on bottom

**Comment by hunicus** [comment]
Nice, reasonable compromise to have it on 1 line in most cases until it absolutely needs to be split.

One gripe: the icon is not properly aligned vertically:

Screenshot from 2023-10-10 00-16-24

This looks to be because of the `viewBox="0 0 300 300"` attribute on the SVG. While the SVG document is 300x300, the image is only 256x256. I suggest either setting `viewBox="0 0 256 256"` or removing the extra space in the SVG document.

Screenshot from 2023-10-10 00-31-58

**Comment by orangesurf** [comment]
Well spotted. Fixed.

image
image

**Comment by hunicus** [comment]
Looks good to me 👍

Now I suggest moving these commits to an `orangesurf/feature` branch on the `mempool/mempool` repo instead of on the master branch of your `orangesurf/mempool` repo. Much easier to review that way (see examples [here](https://github.com/mempool/mempool/branches)).

Then just cherry-pick the 3 commits from this pull request on to your new feature branch. Will probably need to open a new pull request though...just make sure to link to this one for context and I'll ACK that one.

Please make sure all your commits are signed too. Feel free to reach out if you want any guidance.

**Comment by orangesurf** [comment]
moved to https://github.com/mempool/mempool/pull/4327

**Pull request closed without merging**